CONTRIBUTING.md

```diff
@@ -509,28 +498,28 @@ There are a number of ways timeouts can be defined in Akka tests. The following
 
 Special care should be given to `expectNoMessage` calls, which indeed will wait for the entire timeout before continuing. Therefore a shorter timeout should be used in those, for example `200.millis` or `300.millis`. Prefer the method without timeout parameter, which will use the configured `expect-no-message-default` timeout.
 
-You can read up on `remaining` and friends in [TestKit.scala](https://github.com/akka/akka/blob/main/akka-testkit/src/main/scala/akka/testkit/TestKit.scala).
+You can read up on `remaining` and friends in [TestKit.scala](https://github.com/apache/incubator-pekko/blob/main/akka-testkit/src/main/scala/akka/testkit/TestKit.scala).
 
 ### Contributing modules
 
 For external contributions of entire features, the normal way is to establish it
 as a stand-alone project first, to show that there is a need for the feature. If there is enough interested, the
-next step would be to add it to Akka as an "may change"-feature (possibly in a new subproject) and marking it's public api with the `ApiMayChange` annotation,
+next step would be to add it to Pekko as an "may change"-feature (possibly in a new subproject) and marking it's public api with the `ApiMayChange` annotation,
 then when the feature is hardened, well documented and
-tested it becomes an officially supported Akka feature.
+tested it becomes an officially supported Pekko feature.
 
-[List of Akka features marked as may change](https://doc.akka.io/docs/akka/current/common/may-change.html)
+[List of Pekko features marked as may change](https://doc.akka.io/docs/akka/current/common/may-change.html)
```

Comment:
I agree - we should remove any links to akka.io and add the sections back when we have our own website - maybe, we could comment out these sections in the `.md` file to act as a reminder that the sections need to be revived

Reply:
Ok - I will replace all akka.io links with [pekko.apache.org/](https://pekko.apache.org/). Although that is not operational yet, hopefully it will be soon.